scripts/ci/check_compliance.py

```diff
@@ -561,6 +561,36 @@ def parse_kconfig(self, filename="Kconfig", hwm=None):
             # Clean up the temporary directory
             shutil.rmtree(kconfiglib_dir)
 
+    def get_logging_syms(self, kconf):
+        # Returns a set() with the names of the Kconfig symbols generated with
+        # logging template in samples/tests folders. The Kconfig symbols doesn't
+        # include `CONFIG_` and for each module declared 4 Kconfig symbols are
+        # created:
+        #   - *_LOG_LEVEL
+        #   - *_LOG_LEVEL_DBG
+        #   - *_LOG_LEVEL_ERR
+        #   - *_LOG_LEVEL_INF
+        #   - *_LOG_LEVEL_WRN
+
+        # Warning: Needs to work with both --perl-regexp and the 're' module.
+        regex = r"^\s*(?:module\s*=\s*)([A-Z0-9_]+)\s*(?:#|$)"
+
+        # Grep samples/ and tests/ for symbol definitions
+        grep_stdout = git("grep", "-I", "-h", "--perl-regexp", regex, "--",
+                          ":samples", ":tests", cwd=ZEPHYR_BASE)
```

Comment:
Is there a point to the colon at the beginning of the pathspec?

Reply:
No, I copy pasted it from `get_defined_syms`, maybe it help to get that it's a pathspec, I don't know